src/js/components/TaskDetail.js

```diff
@@ -1,24 +1,29 @@
-import classNames from 'classnames';
+import mixin from 'reactjs-mixin';
 /* eslint-disable no-unused-vars */
 import React from 'react';
 /* eslint-enable no-unused-vars */
+import {StoreMixin} from 'mesosphere-shared-reactjs';
 
 import DescriptionList from './DescriptionList';
+import MarathonStore from '../stores/MarathonStore';
 import MesosStateStore from '../stores/MesosStateStore';
 import MesosSummaryStore from '../stores/MesosSummaryStore';
 import ResourceTypes from '../constants/ResourceTypes';
 import RequestErrorMsg from './RequestErrorMsg';
-import SidePanelContents from './SidePanelContents';
+import ServicesBreadcrumb from './ServicesBreadcrumb';
+import PageHeader from './PageHeader';
 import TaskDebugView from './TaskDebugView';
 import TaskDirectoryView from './TaskDirectoryView';
 import TaskDirectoryStore from '../stores/TaskDirectoryStore';
 import TaskStates from '../constants/TaskStates';
-import TaskUtil from '../utils/TaskUtil';
 import Units from '../utils/Units';
 
+import InternalStorageMixin from '../mixins/InternalStorageMixin';
```

Comment:
Sorry, don't get it. :) Do you mean I should place the Mixins into the normal imports?